doc/user/content/overview/architecture.md

```diff
@@ -8,23 +8,97 @@ menu:
     weight: 10
 ---
 
-Materialize is a horizontally scalable, highly available distributed system. It separates storage and compute into different layers, allowing data ingestion, computation, and query serving to be **performed and scaled independently**.
+Materialize is a streaming database designed to provide correct, low latency answers to queries that need to be answered repeatedly by **incrementally maintaining answers** to those queries. Materialize enables users to ingest as much data as they want (by separating compute and storage into decoupled layers), perform as many queries as they want (by being horizontally and vertically scalable), while being highly available and flexible to operate (by offering active-active replication).
+
+
+The Materialize architecture is broken into three separate layers:
+* Storage: which is responsible for reading from and writing to external sources and sinks.
+* Compute: which is responsible for incrementally maintaining the results of user requested queries.
+* SQL Control Plane (Adapter): which is responsible for actually responding to user requests, and serving results back to users.
+
+Each of these components perform their work independently and can be **scaled independently**.
+
+This page will describe the responsibilities of these three layers, and how users can interact with them in SQL.
 
 {{<
     figure src="/images/architecture_overview.jpg"
     alt="Materialize architecture overview"
     width="700"
 >}}
 
-## Logical components
+## Regions
+
+All of the components of the Materialize platform are scoped to a single AWS region and cannot be shared between regions. [TODO: is this true?] This is important because we'll soon talk about sharing resources and its important to remember that we cannot share anything across regions.
```

Comment:
i still don't know if this is true/this section needs to be reworked to indicate that everything is scoped to (iiuc) user + region

Reply:
I just removed this, as it seemed like more of an implementation detail